src/en/components/card/base.md

```diff
@@ -0,0 +1,21 @@
+---
+github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-card
+figma: https://www.figma.com/community/file/1128687821123298228
+permalink: false
+tags: ['cardEN', 'header']
+---
+
+# Card <br>`<gcds-card>`
+
+_Also called: tile, content card._
+
+A card is a box containing structured, actionable content on a single topic. 
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Card component preview" %}
+<gcds-card card-title="Title of the article" tag="Tag" href="#" description="This is a description">
```

Comment:
Could you please update the content to be consistent with what's in the anatomy here:

https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/branch/LXmH6TsKGdG1lEVO7O1Qv6/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=7628-11006&mode=design&t=hpk1GYNMrHVqdHne-0

Aka `card-title="Card title link"` `description="Description or supporting text relating to the headline. Longer text will be truncated with ..."`